stubs/seaborn/seaborn/regression.pyi

```diff
@@ -25,15 +28,15 @@ def lmplot(
     height: float = 5,
     aspect: float = 1,
     markers: str = "o",
-    sharex: bool | Literal["col", "row"] | None = None,
-    sharey: bool | Literal["col", "row"] | None = None,
+    sharex: bool | Literal["col", "row"] | None = None,  # deprecated
+    sharey: bool | Literal["col", "row"] | None = None,  # deprecated
```

Comment:
https://github.com/mwaskom/seaborn/blob/d4b8de81f74a153bb3c15af760d956d32aad980b/seaborn/regression.py#L601-L603. I'll handle the deprecations of seaborn using `typing_extensions.deprecated` in another PR.